templates/python-selenium/README.md

```diff
@@ -1,5 +1,20 @@
-# Selenium Actor template
+# Python - Selenium & Chrome Actor template
 
-The `README.md` file documents what your actor does and how to use it, which is then displayed in the Console or Apify Store. It's always a good idea to write a `README.md`. In a few months, not even you will remember all the details about the actor.
+A template example built with Selenium and headless Chrome browser to scrape a website and save the results to storage. The URL of the web page is passed in via input, which is defined by the [input schema](https://docs.apify.com/platform/actors/development/input-schema). The template uses the [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) to load and process the page. Enqueued URLs are stored in the default [request queue](https://docs.apify.com/sdk/python/reference/class/RequestQueue). The data are then stored in the default [dataset](https://docs.apify.com/platform/storage/dataset) where you can easily access them.
 
-You can use [Markdown](https://www.markdownguide.org/cheat-sheet) language for rich formatting.
+## Included features
+
+- **[Apify SDK](https://docs.apify.com/sdk/python/)** - toolkit for building Apify Actors
+- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your Actor's input
+- **[Request queue](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-request-queues)** - queues into which you can put the URLs you want to scrape
+- **[Dataset](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-datasets)** - store structured data where each object stored has the same attributes
+
+## How it works
+This code is a Python script that uses the Apify platform to scrape web pages and extract data from them. Here's a brief overview of how it works:
```

Comment:
This is not really true, the script doesn't use the Apify platform if you run it locally. I would replace "Apify platform" with "Selenium" here.